modules/aws/s3.go

```diff
@@ -262,33 +262,34 @@ func EmptyS3BucketE(t testing.TestingT, region string, name string) error {
 }
 
 // GetS3BucketLoggingTarget fetches the given bucket's logging configuration status and returns it as a string
-func GetS3BucketLoggingTarget(t testing.TestingT, awsRegion string, bucket string) string {
-	loggingTarget, err := GetS3BucketLoggingTargetE(t, awsRegion, bucket)
+func GetS3BucketLoggingTarget(t testing.TestingT, awsRegion string, bucket string) (string, string) {
+	loggingTarget, loggingObjectTargetPrefix, err := GetS3BucketLoggingTargetE(t, awsRegion, bucket)
 	require.NoError(t, err)
 
-	return loggingTarget
+	return loggingTarget, loggingObjectTargetPrefix
 }
 
-// GetS3BucketLoggingE fetches the given bucket's versioning configuration status and returns it as a string
-func GetS3BucketLoggingTargetE(t testing.TestingT, awsRegion string, bucket string) (string, error) {
+// GetS3BucketLoggingE fetches the given bucket's logging configuration status and returns it as the following strings:
+// `TargetBucket` and `TargetPrefix` of the `LoggingEnabled` property for an S3 bucket
+func GetS3BucketLoggingTargetE(t testing.TestingT, awsRegion string, bucket string) (string, string, error) {
 	s3Client, err := NewS3ClientE(t, awsRegion)
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
 
 	res, err := s3Client.GetBucketLogging(&s3.GetBucketLoggingInput{
 		Bucket: &bucket,
 	})
 
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
 
 	if res.LoggingEnabled == nil {
-		return "", S3AccessLoggingNotEnabledErr{bucket, awsRegion}
+		return "", "", S3AccessLoggingNotEnabledErr{bucket, awsRegion}
 	}
 
-	return aws.StringValue(res.LoggingEnabled.TargetBucket), nil
+	return aws.StringValue(res.LoggingEnabled.TargetBucket), aws.StringValue(res.LoggingEnabled.TargetPrefix), nil
```

Comment:
To maintain backwards compatibility, should there be a separate method to return just the target prefix? Or possibly the bucket + prefix?

Reply:
+1 to introduce a separate method that just retrieves the target prefix.

Reply:
That's a really good point! Thanks a lot both 🙂

Reply:
So in terms of backwards compatibility - did you also mean to keep the naming of the method as it was originally? It was quite a generic name, so there's now 2 functions: `GetS3BucketLoggingTarget` & `GetS3BucketLoggingTargetPrefix`

Reply:
Yea, I think preserving the old name makes sense too. It's not an ideal name, but probably not bad enough it merits a backwards incompatible change.

Reply:
Ok, will change that too - thanks, Jim!